ensime_shared/spec/features/suggestion_format.feature

```diff
@@ -13,22 +13,6 @@ Feature: Format a Completion Suggestion
     | <byname>[ByNameType]   | => ByNameType |
     | <repeated>[VarargType] | VarargType*   |
 
-  Scenario: Format single type parameter
-    Given A list of type parameters:
-      | tparam |
-      | A      |
-    When We concat the type parameters
-    Then We get the format [A]
-
-  Scenario: Format multiple type parameters
-    Given A list of type parameters:
-      | tparam |
-      | A      |
-      | B      |
-      | C      |
-    When We concat the type parameters
-    Then We get the format [A, B, C]
```

Comment:
I was planning to add type parameters to signatures in completion popup but it needs some extra work in the server. I've created #276 to track it. No worries, I can bring this function back if needed.